### PR TITLE
Explicitly install 'which' into f22 nodepool images

### PR DESCRIPTION
**Pull request by seandst**

### CODE REVIEW

File: ci/nodepool/scripts/prepare_node.sh [hunk 1]
```diff
@@ -40,6 +40,8 @@ elif  [ "${DISTRIBUTION}" == "fedora" ]; then
     sudo "${PKG_MGR}" install -y python-dnf
     if  [ "${DISTRIBUTION_MAJOR_VERSION}" == "22" ]; then
         sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-22.noarch.rpm
+        # "which" isn't installed in fedora 22 by default?
```
Comment (elyezer) [inline ci/nodepool/scripts/prepare_node.sh]: This is really weird! But LGTM

Comment (seandst) [inline ci/nodepool/scripts/prepare_node.sh]: Yeah, I cheated with this one (as I sometimes do...) and it's already applied on the nodepool box and fixed up some weird pulp-smash errors complaining about "which" not being installed.

Comment (elyezer) [inline ci/nodepool/scripts/prepare_node.sh]: Thank you very much.

Comment (seandst) [inline ci/nodepool/scripts/prepare_node.sh]: > it's already applied on the nodepool box

I've pulled down this commit to the nodepool box.

**Pull request merged**